docs/README.md

```diff
@@ -7,6 +7,12 @@ To start the docs site in development mode, from the project root, run:
 ```sh
 yarn && yarn start
 ```
+If you do not have yarn installed, select your OS and follow the instructions on the [Yarn website](https://yarnpkg.com/lang/en/docs/install/#mac-stable).
+
+*Mac users: note that you must have version 10.10.3 or higher for the Yarn installation to work as of 2019*
```

Comment:
Do we need this warning?

Reply:
I personally didn't have the latest version of mac OS, so for me, it was a significant issue that I had to fix for my tests to run, and I couldn't really find much documentation on what the solution to the problem was, considering I didn't have enough storage to update my OS.

I think it could be a useful warning for people not familiar with Yarn and the installation process in general, but I can only speak for myself, so your call.

Reply:
Thanks for providing more detail. This would be better listed in yarn documentation.